src/hub_load/download-packages.sh

```diff
@@ -13,11 +13,10 @@ mkdir -p $WORKDIR/jars
 # Otherwise, this script will pull down a jars.zip from S3 and unpack it to populate the jars directory
 # on the image by running this script in the docker build
 #
-if [ ! -d jars ]; then
```

Comment:
The purpose of the _if_ here is to avoid re-downloading if the jars dir is already present. Since the jars collection is large thought this would be a nice optimization, but if it's breaking things I see no issue in removing it.